### ISSUE TITLE
длинный посткоммент к сенс или что-то из этого сабэнтри?

### ISSUE THREAD

**Issue by solanum-tuberosum** [obelyaev/abaevdict#95]
![image](https://user-images.githubusercontent.com/13102259/75432004-cbd8c380-595e-11ea-8c83-5b0218431595.png)
Это длинный посткоммент к сенс или что-то из этого пресабэнтри?..

**Comment by obelyaev** [comment]
Сложный вопрос. По идее — да: он объясняет употребление слова и одновременно приводит композиты, в которых оно встречается. По идее всё, что начинается со "ср. cæskom" — это сабэнтри. Но легко ли будет всё это отформатировать так, как надо?.. Как Вы думаете, хватает ли наших возможностей?

**Comment by jvsntsn** [comment]
![image](https://user-images.githubusercontent.com/48504970/77153927-ca0ca680-6aab-11ea-97d8-85628f3a9acd.png)
мой пример явно не такой масштабный, как Ирин, но вот тут запихнуть все в definition, или попытаться что-то сделать с осетинскими словами?

**Comment by solanum-tuberosum** [comment]
> ![image](https://user-images.githubusercontent.com/48504970/77153927-ca0ca680-6aab-11ea-97d8-85628f3a9acd.png)
> мой пример явно не такой масштабный, как Ирин, но вот тут запихнуть все в definition, или попытаться что-то сделать с осетинскими словами?

Я бы сказала что все в definition, что до слов "варианты", а начиная с "вариантов" запихнула бы как postcomment к sensegroup, но хз

**Comment by solanum-tuberosum** [comment]
> Сложный вопрос. По идее — да: он объясняет употребление слова и одновременно приводит композиты, в которых оно встречается. По идее всё, что начинается со "ср. cæskom" — это сабэнтри. Но легко ли будет всё это отформатировать так, как надо?.. Как Вы думаете, хватает ли наших возможностей?

Вроде у меня получилось... Только два момента
![image](https://user-images.githubusercontent.com/13102259/78468958-2666fb80-7725-11ea-8dcc-104f9f30e8c1.png)
* последняя пресабэнтри груп отделена от всех тире. я поставила пока тире в прекоммент, но перед ним все равно показывается точка с запятой, а не точка.
![image](https://user-images.githubusercontent.com/13102259/78468978-50202280-7725-11ea-9403-7541166ed763.png)
* последний посткоммент добавила к последней пресабэнтри груп, но по логике это комментарий к значениям самого слова. но наш посткоммент к лемме ставится после примеров. что делать? забить?

**Comment by obelyaev** [comment]
> > ![image](https://user-images.githubusercontent.com/48504970/77153927-ca0ca680-6aab-11ea-97d8-85628f3a9acd.png)
> > мой пример явно не такой масштабный, как Ирин, но вот тут запихнуть все в definition, или попытаться что-то сделать с осетинскими словами?
> 
> Я бы сказала что все в definition, что до слов "варианты", а начиная с "вариантов" запихнула бы как postcomment к sensegroup, но хз

А что даст запихивание в посткоммент? Всё равно эти слова не будут индексироваться как значимые элементы. А по идее это всё варианты головного слова.

**Comment by obelyaev** [comment]
> > Сложный вопрос. По идее — да: он объясняет употребление слова и одновременно приводит композиты, в которых оно встречается. По идее всё, что начинается со "ср. cæskom" — это сабэнтри. Но легко ли будет всё это отформатировать так, как надо?.. Как Вы думаете, хватает ли наших возможностей?
> 
> Вроде у меня получилось... Только два момента
> ![image](https://user-images.githubusercontent.com/13102259/78468958-2666fb80-7725-11ea-8dcc-104f9f30e8c1.png)
> 
>     * последняя пресабэнтри груп отделена от всех тире. я поставила пока тире в прекоммент, но перед ним все равно показывается точка с запятой, а не точка.
>       ![image](https://user-images.githubusercontent.com/13102259/78468978-50202280-7725-11ea-9403-7541166ed763.png)
> 
>     * последний посткоммент добавила к последней пресабэнтри груп, но по логике это комментарий к значениям самого слова. но наш посткоммент к лемме ставится после примеров. что делать? забить?

Да, это сложный случай. Можно придумать что-нибудь, чтобы правильно отображалось, но меня скорее волнует, что последний комментарий оказывается PostComment к Subentry, хотя по сути здесь Абаев комментирует последовательно значения: сначала "рот" (и даёт информацию об идиомах и т.д.), затем и "ущелье". Т.е. это такое дистантное расположение посткомментов, на самом деле… (Такое ни один язык разметки не потянет: поэтому мы и сделали систему с ручным проставлением номеров значений и примеров, например.) По логике тут надо бы делать Subentry внутри PostComment, но это как-то не очень… Есть какие-нибудь идеи?

**Comment by solanum-tuberosum** [comment]
![image](https://user-images.githubusercontent.com/13102259/89021875-e22de580-d329-11ea-9c20-20e30459db68.png)
Вот еще, здесь я пока все положила в посткоммент
